app/webpacker/src/stylesheets/_tables.scss

```diff
@@ -118,7 +118,7 @@ table.small-table {
       td:nth-of-type(4) { width: 9% }
```

Comment:
What happened to the option of removing all of these?

Reply:
also, the tables are going to have massive restiling, that's a WIP